tools/pinning/oldest/pyproject.toml

```diff
@@ -36,84 +36,59 @@ acme = {path = "../../../acme", extras = ["test"]}
 
 # Oldest dependencies
 # We specify the oldest versions our dependencies that we're trying to keep
-# support for below. Usually these version numbers are taken from the packages
-# of our dependencies available in popular LTS Linux distros. Keeping
-# compatibility with those versions makes it much easier for OS maintainers to
-# update their Certbot packages.
+# support for below. We should only update these packages as needed to make use
+# of features in newer versions of our dependencies. Keeping compatibility with
+# older packages makes it much easier for OS maintainers to update their
+# Certbot packages if needed or desired.
 #
 # When updating these dependencies, we should ideally try to only update them
```

Comment:
```suggestion
# When updating these dependencies, we should update them no further than
```

Reply:
I asked you about this at https://opensource.eff.org/eff-open-source/pl/ifqm9k656tdkfft3zqgfzosmqh and I thought you were OK with the current language? Did you change your mind?

I personally still prefer the softer language as doing what's easiest for us here is a pretty strong motivator to me right now.

Reply:
We can keep that; I was trying to improve clarity here, and cut the rest in the process of making the sentence cleaner to read. If we don't have that implication about "ideally try to" elsewhere, this isn't the worst spot for it.

What about "When updating these dependencies, we should try to update them no further than the oldest version of the dependency found in CentOS/RHEL 8 + EPEL (or newer versions of CentOS/RHEL + EPEL) as our Certbot packages there see frequent updates."?

Reply:
That works for me.